pkg/vz/vz_driver_darwin.go

```diff
@@ -130,6 +130,7 @@ func (l *LimaVzDriver) Validate() error {
 			"Lima",
 			"Socket",
 			"MACAddress",
+			"Metric",
 			"Interface",
 		); len(unknown) > 0 {
 			logrus.Warnf("vmType %s: ignoring networks[%d]: %+v", *l.Instance.Config.VMType, i, unknown)
```

Comment:
It is easy to grep for the warning text and understand why it happens but the warning should make it clear that this is a validation warning, and the issue is unknown property. Maybe something like:

    validate vmType vz: ignoring unknown property networks[1]: "Metric"

Reply:
SGTM

If you are going to work on this, you should also take a look at the corresponding code in the `pkg/wsl2/wsl_driver_windows.go` file to try to keep things in sync (I noticed vz uses `ignoring` while `wsl2` uses `Ignoring`, which caused me to miss them on my initial grepping).

Reply:
Of course validation is going to be a big rabbit hole; look at #2512 for more thoughts on how it should be refactored.

Personally I don't want to touch it (validation) until I had a chance to replace the current `FillDefaults` with the upcoming code from template assembly because that might change things around anyways.

Reply:
I don't have time for this now, added 3011